Incompatible with Canvas Renderer

Minecraft Version: 1.18.1
Canvas Version: 1.0.2281
Bits And Chisels Version: 2.6.2
Other Mods:  fabric-api-0.45.0


Crash: 
[Render thread/FATAL]: Unreported exception thrown!
java.lang.NoClassDefFoundError: io/vram/frex/compat/fabric/FabricMaterial
	at io.github.coolmineman.bitsandchisels.CanvasHelper.getMaterial(CanvasHelper.java:23) ~[bitsandchisels-2.6.2.jar:?]
	at net.minecraft.class_2680.recomputeIfNeeded(class_2680.java:555) ~[client-intermediary.jar:?]
	at net.minecraft.class_2680.getQuads(class_2680.java:566) ~[client-intermediary.jar:?]
	at io.github.coolmineman.bitsandchisels.BitMeshes.doQuad(BitMeshes.java:161) ~[bitsandchisels-2.6.2.jar:?]
	at io.github.coolmineman.bitsandchisels.BitMeshes.createMesh(BitMeshes.java:73) ~[bitsandchisels-2.6.2.jar:?]
	at io.github.coolmineman.bitsandchisels.BitsBlockEntity.rebuildMesh(BitsBlockEntity.java:124) ~[bitsandchisels-2.6.2.jar:?]
	at io.github.coolmineman.bitsandchisels.BitsBlockEntity.postFromClientTag(BitsBlockEntity.java:128) ~[bitsandchisels-2.6.2.jar:?]
	at io.github.coolmineman.bitsandchisels.BitsBlockEntity.method_11014(BitsBlockEntity.java:63) ~[bitsandchisels-2.6.2.jar:?]
	at net.minecraft.class_634.method_38542(class_634.java:1077) ~[client-intermediary.jar:?]
	at java.util.Optional.ifPresent(Optional.java:178) ~[?:?]
	at net.minecraft.class_634.method_11094(class_634.java:1074) ~[client-intermediary.jar:?]
	at net.minecraft.class_2622.method_11292(class_2622.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.class_2622.method_11054(class_2622.java:14) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:21) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:151) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:125) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:110) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1080) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:733) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:238) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.12.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.12.jar:?]
Caused by: java.lang.ClassNotFoundException: io.vram.frex.compat.fabric.FabricMaterial
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:158) ~[fabric-loader-0.12.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	... 28 more

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,6 @@ build
 .brachyura
 run
 netbeans
+.idea
+main.iml
+out

--- a/buildscript/src/main/java/Buildscript.java
+++ b/buildscript/src/main/java/Buildscript.java
@@ -50,7 +50,7 @@ public class Buildscript extends FabricProject {
         d.addMaven("https://storage.googleapis.com/devan-maven/", new MavenId("net.devtech:Stacc:1.2.3"), ModDependencyFlag.RUNTIME, ModDependencyFlag.COMPILE, ModDependencyFlag.JIJ);
         // Compat
         d.addMaven("https://maven.shedaniel.me/", new MavenId("me.shedaniel:RoughlyEnoughItems-api-fabric:6.0.247-alpha"), ModDependencyFlag.COMPILE);
-        d.addMaven("https://maven.vram.io", new MavenId("io.vram:frex-fabric-mc117:6.0.46"), ModDependencyFlag.COMPILE);
+        d.addMaven("https://maven.vram.io", new MavenId("io.vram:frex-fabric-mc118:6.0.236"), ModDependencyFlag.COMPILE);
         d.addMaven("https://oskarstrom.net/maven", new MavenId("net.oskarstrom:DashLoader:2.0"), ModDependencyFlag.COMPILE);
     }
 

--- a/src/main/java/io/github/coolmineman/bitsandchisels/CanvasHelper.java
+++ b/src/main/java/io/github/coolmineman/bitsandchisels/CanvasHelper.java
@@ -4,7 +4,7 @@ import io.vram.frex.api.material.MaterialFinder;
 import io.vram.frex.api.material.MaterialMap;
 import io.vram.frex.api.renderer.Renderer;
 import io.vram.frex.api.rendertype.RenderTypeUtil;
-import io.vram.frex.compat.fabric.FabricMaterial;
+import io.vram.frex.fabric.compat.FabricMaterial;
 import net.fabricmc.fabric.api.renderer.v1.material.RenderMaterial;
 import net.minecraft.block.BlockState;
 import net.minecraft.client.render.RenderLayers;
@@ -14,9 +14,9 @@ public class CanvasHelper {
     private CanvasHelper() { }
 
     public static final RenderMaterial getMaterial(BlockState state, Sprite sprite) {
-        io.vram.frex.api.material.RenderMaterial material = (io.vram.frex.api.material.RenderMaterial) MaterialMap.get(state).getMapped(sprite);
+        io.vram.frex.api.material.RenderMaterial material = MaterialMap.get(state).getMapped(sprite);
         if (material != null) {
-            MaterialFinder finder = Renderer.get().materialFinder().copyFrom(material);
+            MaterialFinder finder = Renderer.get().materials().materialFinder().copyFrom(material);
             RenderTypeUtil.toMaterialFinder(finder, RenderLayers.getBlockLayer(state));
             return FabricMaterial.of(finder.find());
         } else {

--- a/src/main/java/io/github/coolmineman/bitsandchisels/CanvasHelper.java
+++ b/src/main/java/io/github/coolmineman/bitsandchisels/CanvasHelper.java
@@ -13,7 +13,7 @@ import net.minecraft.client.texture.Sprite;
 public class CanvasHelper {
     private CanvasHelper() { }
 
-    public static final RenderMaterial getMaterial(BlockState state, Sprite sprite) {
+    public static RenderMaterial getMaterial(BlockState state, Sprite sprite) {
         io.vram.frex.api.material.RenderMaterial material = MaterialMap.get(state).getMapped(sprite);
         if (material != null) {
             MaterialFinder finder = Renderer.get().materials().materialFinder().copyFrom(material);

